utils/mt5311/Makefile

```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mt5311
+PKG_VERSION:=238-2cb1304
```

Comment:
Since you are using sources from GitHub and checking this out, this is not necessary.

Reply:
Was used to add the number to commits before the short commit hash by other packaging, but seems unusual for OpenWrt packages.

Removed it.